src/commands/publish.yml

```diff
@@ -15,14 +15,51 @@ parameters:
   access:
     description: tells the registry whether this package should be published as public or restricted
     default: public
-    type: string
+    type: enum
+    enum: [public, restricted]
+  use_tarfile:
```

Comment:
I think I would rather just have this be `tarfile` that is a string. Then pass in `TARPATH` that is output from the trust plugin. That will protect against renames in the future and also make the command more flexible.

Reply:
@amphro as far as I know there's no way to pass information from one command to the next, which is why I did it this way

Reply:
Dang, ok.